### DISCUSSION TITLE
Serving commands over wifi, possible issue with documentation

### DISCUSSION THREAD

**Discussion by SturdyFool10** [PlummersSoftwareLLC/NightDriverStrip#185] [Q&A]
I am having a bit of trouble sending data to the nightdriver strip using the included documentation, if a couple of unknowns could be answered it would go a long way, namely, what protocol should be used? I tired UDP to no avail and I am wondering if there were any specific frameworks you would use to accomplish this

**Comment by davepl** [comment]
We’ve got a C# implementation that sends data to the client strips, and you can actually do it in anything - python, C, whatever.   It also supports compression.  Basically you open a TCP socket on port 49152 and then start sending packets.

You cold reverse engineer what the packet expects by looking at the function ProcessIncomingData, but it’d be easier if I could just give you the code, so let me talk to Rutger about that and we’ll see if we can’t share it.

- Dave

**Reply by SturdyFool10** [reply]
ok, thank you, I tried UDP since I had no direction but this will help greatly
